pkg/pfcpsim/pfcpsim.go

```diff
@@ -92,6 +94,19 @@ func (c *PFCPClient) sendMsg(msg message.Message) error {
 		return err
 	}
 
+	if c.remoteAddr != "" {
+		rAddr, err := net.ResolveUDPAddr("udp", c.remoteAddr)
+		if err != nil {
+			return err
+		}
+
+		if _, err := c.conn.WriteToUDP(b, rAddr); err != nil {
+			return err
+		}
+
+		return nil
+	}
+
 	if _, err := c.conn.Write(b); err != nil {
 		return err
 	}
```

Comment:
What about using always `WriteToUDP` method, for consistency? When dialing up a UDP address we save the remote address.